sdk/src/main/java/com/bugsnag/android/SessionTracker.java

```diff
@@ -64,6 +64,11 @@
      * @param user the session user (if any)
      */
     void startNewSession(@NonNull Date date, @Nullable User user, boolean autoCaptured) {
+        if (configuration.getSessionEndpoint() == null) {
+            Logger.warn("The session tracking endpoint has not been set. "
```

Comment:
Should have mentioned this earlier, sorry - Should this warning be a one-time thing? We could end up adding a lot of logger noise.

Reply:
The `Logger` class will automatically disable itself in production if that's a concern? The default implementation of session tracking waits at least 30 seconds before starting a new session on Android, so for the default use case I'd imagine this message would only be logged once or twice.

Reply:
I guess that's ok, are there any similar cases already present in the notifier?

Reply:
@Cawllec it's not particularly easy to test log messages in Android unfortunately